testing/features/step_definitions/navigation_steps.rb

```diff
@@ -0,0 +1,70 @@
+# frozen_string_literal: true
+
+Given("the navigation is on the page") do
+  @component = Navigation::Standard.new.tap(&:load)
+end
+
+Given("the navigation is on the page with header links") do
+  @component = Navigation::WithHeader.new.tap(&:load)
+end
+
+Given("the navigation does not fit on the screen") do
```

Comment:
Everywhere this and the other resize logic is used. If you just use a tag of `@small_screen` it will work for both mobiles and non mobiles, and saves you a load of steps.

Reply:
@luke-hill Not sure about implementing this.  It litters the gherkin with `@small_screen` to remove a single step definition.  The `Given` clauses will obviously need to stay in the gherkin as it is part of the scenario.  Using  the `@small_screen` tag with `Background:` instead of `Scenario:` produces an obscure error: 

```
When using the Selenium Logger that is set to pipe to a file, the Net::HTTP adapter (default),
can return unencoded binary (confusingly called ASCII-8BIT in Ruby).
Consequently we set the logger to binmode so it doesn't try to encode the data - this would always
cause errors for non-ASCII characters, whatever the parent encoding is. An example of this is ©.
```

Reply:
Fair enough. i think this is one we can table to fix later.

Ordinarily I'd tell you to tag the rule - Which then saves you a whole heap of lines - However you can't do this (Yet).

The only issue with this is this logic will fail on mobiles. So if you check in `hooks.rb` I'm happy if you want to copy the conditional there, and I can make a tech debt ticket to refactor this feature later on once we've done the upgrades required.

Reply:
I don't know what this means.  What logic? Resizing the screen?  Are you saying I need to tag these with `@not_mobile`?

Reply:
Ah sorry. No not quite.

So if you see here: https://github.com/citizensadvice/design-system/blob/master/testing/features/support/hooks.rb#L12-L13 - You'll see the logic that would be desirable.

But for now, as a short-term fix. Just whacking a `unless mobile_phone?` at the end of your resizing logic should be fine. I'll tidy everything up down the line once the requisite updates are done.

Reply:
Great, thanks.